Update scheduler content with Slim CI jobs (#3380)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/dbt-cloud-job.md

title: "Deploy with dbt Cloud"
id: "dbt-cloud-job"
description: "You can enable continuous integration (CI) to test every single change prior to deploying the code to production just like in a software development workflow."
hide_table_of_contents: true
tags: ["scheduler"]
---

dbt Cloud offers the easiest way to run your dbt project in production. [Deploying](docs/deploy/deployments) with dbt Cloud lets you:
- Keep production data fresh on a timely basis
- Ensure CI and production pipelines are efficient
- Identify the root cause of failures in deployment environments
- Maintain high quality code and data in production
- Gain visibility into the health of deployment jobs, models, and tests

> Learn more about the features you can use in dbt Cloud to help your team ship timely, quality production data easier.

Reviewer: ```suggestion
Learn more about the features you can use in dbt Cloud to help your team ship timely, quality production data more easily.
```

<div className="grid--2-col">

<Card
    title="Artifacts"
    body="dbt Cloud generates and saves artifacts for your project, which it uses to power features like creating docs for your project and reporting the freshness of your sources."
    link="/docs/deploy/artifacts"
    icon="pencil-paper"/>

<Card
    title="Job scheduler"
    body="The job scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production environments."
    link="/docs/deploy/job-scheduler"
    icon="pencil-paper"/>

<Card
    title="Job settings"
    body="Create and schedule jobs for the dbt Cloud scheduler to run."
    link="/docs/deploy/job-settings"
    icon="pencil-paper"/>

<Card
    title="Job commands"
    body="Configure which dbt commands to execute when running a dbt job."
    link="/docs/deploy/job-commands"
    icon="pencil-paper"/>

<Card
    title="Job triggers"
    body="Set up a cron-based schedule or an event-driven trigger by API or on pull requests for CI jobs."
    link="/docs/deploy/job-triggers"
    icon="pencil-paper"/>

<Card
    title="Job notifications"

>     body="Receive email or Slack channel notifications when a job run succeeds, fails, or is cancelled so you can respond quickly and begin remediation if necessary."

Reviewer: ```suggestion
    body="Receive email or Slack channel notifications when a job run succeeds, fails, or is canceled so you can respond quickly and begin remediation if necessary."
```

    link="/docs/deploy/job-notifications"
    icon="pencil-paper"/>

<Card
    title="Run visibility"
    body="View the history of your runs and the model timing dashboard to help identify where improvements can be made to the scheduled jobs."
    link="/docs/deploy/run-visibility"
    icon="pencil-paper"/>

<Card
    title="Source freshness"
    body="Enable snapshots to capture the freshness of your data sources and configure how frequent these snapshots should be taken. This can help you determine whether your source data freshness is meeting your SLAs."
    link="/docs/deploy/source-freshness"
    icon="pencil-paper"/>

<Card
    title="Webhooks for your jobs"
    body="Create outbound webhooks to send events about your dbt jobs' statuses to other systems in your organization."
    link="/docs/deploy/webhooks"
    icon="pencil-paper"/>

<Card
    title="Dashboard status tiles"
    body="Set up status tiles to see the data freshness and quality checks whenever you view your data. "
    link="/docs/deploy/dashboard-status-tiles"
    icon="pencil-paper"/>

<Card
    title="Continuous integration"

>     body="Set up Slim CI checks so you can build and test any modified code in a staging environment when you open PRs and push new commits to your dbt repository."

Reviewer 2: Informal and may not translate well down the road.
```suggestion
    body="Set up Slim CI checks so you can build and test any modified code in a staging environment when you open pull requests and push new commits to your dbt repository."
```

Refer to [build and view your docs](/docs/collaborate/build-and-view-your-docs) for more info.

### Access logs

You can view or download in-progress and historical logs for your dbt runs, making it easy for anyone on the team to debug errors more efficiently.

<Lightbox src="/img/docs/dbt-cloud/deployment/access-logs.jpg" width="85%" title="Access logs for run steps" />

### Model timing 
> Available on [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) dbt Cloud accounts on the [Team or Enterprise plans](https://www.getdbt.com/pricing/).

The model timing dashboard on dbt Cloud displays the composition, order, and time taken by each model in a job run. The visualization appears for successful jobs and highlights the top 1% model durations. This helps you identify bottlenecks in your runs, so you can investigate them and potentially make changes to improve their performance. 

You can find the dashboard on the **Run Overview** page. 

<Lightbox src="/img/docs/dbt-cloud/model-timing.jpg" width="85%" title="The model timing tab displays the top 1% of model durations and visualizes model bottlenecks" />

> 

Author: we may need to create a new page for this "Model timing" section

Reviewer: Yes! let's create a new section for "Run Visibility". Can we put the model timing and [run history](https://docs.getdbt.com/docs/deploy/dbt-cloud-job#run-history) sections in there. Run history is new since you opened this PR.

> When you [deploy dbt jobs](docs/deploy/deployments) with dbt Cloud, you are using the built-in job scheduler. This eliminates the need to manually run dbt commands from the command line or set up an orchestration system. By design, the job scheduler helps streamline your dbt project deployments and runs, keeps your data pipelines functioning seamlessly and efficiently, governs data through standardized practices for making code changes, automates your dbt jobs, and enables teams to get results faster with less effort.

Reviewer: ```suggestion
dbt Cloud offers the easiest way to run your dbt project in production. Deploying with dbt Cloud allows you to:
- Keep production data fresh on a timely basis
- Ensure CI and production pipelines are efficient
- Identify the root cause of failures in deployment environments
- Maintain high quality code and data in production
- Gain visibility into the health of deployment jobs, models, and tests
```

> Learn about more features you can use in dbt Cloud that can help you deploy your jobs easier, faster, and also to monitor those jobs for further fine-tuning. 

Reviewer: ```suggestion
Learn about more about the features you can use in dbt Cloud to help your team ship timely, quality production data with ease.
```

>     body="Use the job scheduler so you can optimize your data transformation workflows, deploy dbt jobs easier, automate your dbt jobs, make code changes in a standardized and governed way, and more."

Reviewer: ```suggestion
    body="The Scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts."
```

>     body="Set up triggers in dbt Cloud so it knows when to run your dbt job and how to run it. You can trigger a job by setting a schedule for it, through Git PRs using Slim CI, or with an API call."

Reviewer: ```suggestion
    body="Set up a cron-based schedule or an event-driven trigger by API or on pull request for CI jobs."
```

>     body="Set up to receive email or Slack channel notifications when a job run succeeds, fails, or is cancelled so you can respond to it quickly and proactively. "

Reviewer: ```suggestion
    body="Receive email or Slack channel notifications when a job run succeeds, fails, or is canceled so you can respond quickly and begin remediation if necessary. "
```

>     body="Create outbound webhooks to send notifications about your dbt jobs to other systems in your organization. This helps trigger automation workflows you have set up."

Reviewer: ```suggestion
    body="Create outbound webhooks to send events about your dbt jobs' statuses to other systems in your organization."
```

>     body="Set up status tiles so you can get quick success/fail updates about your checks in the Dashboard, checks like data freshness and data quality. "

Reviewer: ```suggestion
    body="Set up status tiles so you see data freshness and quality checks directly wherever you view your data. "
```

>     title="Slim CI job"

Reviewer: ```suggestion
    title="Continuous Integration"
```

>     body="Set up Slim CI so you can trigger dbt Cloud to run your job when you open PRs and push new commits."

Reviewer: ```suggestion
    body="Set up Slim CI checks so you can build and test any modified code in a staging environment when you open PRs and push new commits to your dbt repository."
```
